doc/installation.rst

```diff
@@ -62,24 +62,24 @@ The latest official release of DeepQMC can be installed from the Python Package
 Developing
 ----------
 
-In order to have access to the source code and stay up-to-date with the latest developements, DeepQMC can be installed directly from the https://github.com/deepqmc/deepqmc GitHub repository. 
+In order to have access to the source code and stay up-to-date with the latest developments, DeepQMC can be installed directly from the https://github.com/deepqmc/deepqmc GitHub repository. 
 
 To install DeepQMC from the Git repository run::
 
     $ git clone https://github.com/deepqmc/deepqmc
     $ cd deepqmc
     $ pip install -e .[dev] 
 
-Note that the ``-e`` option installs the repository in editable mode and the ``.[dev]`` specification includes the optional dependencies for developement.
+Note that the ``-e`` option installs the repository in editable mode and the ``.[dev]`` specification includes the optional dependencies for development.
 
 If `Pip <https://pip.pypa.io/en/stable/quickstart/>`_ complains about ``setup.py`` not being found, please update to the latest version.
 
-The above installation will result in the CPU version of JAX. However, running DeepQMC on the GPU is highly recomended. To enable GPU support make sure to upgrade JAX to match the CUDA and cuDNN versions of your system. For most users this can be achieved with::
+The above installation will result in the CPU version of JAX. However, running DeepQMC on the GPU is highly recommended. To enable GPU support make sure to upgrade JAX to match the CUDA and cuDNN versions of your system. For most users this can be achieved with::
 
     $ # CUDA 12 installation
     $ pip install --upgrade "jax[cuda12_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
 
     $ # CUDA 11 installation
     $ pip install --upgrade "jax[cuda11_pip]" -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
 
-If issues arise during the installation of JAX, please visit the `JAX Install Guide <https://github.com/google/jax#installation>`_.
+If issues arise during the installation of JAX, or if errors related to CUDA or cuDNN occur at runtime, please visit the `JAX Install Guide <https://github.com/google/jax#installation>`_.
```

Comment:
Sometimes, the problems with JAX installation appear only at runtime, so I've added a mention about that. This a fairly common issue in my experience with JAX